rdr_service/api/genomic_api.py

```diff
@@ -500,12 +502,17 @@ def get_scheduling(self):
         if not participant_id and not start_date:
             raise BadRequest('Participant ID or Start Date parameter is required for use with GenomicScheduling API.')
 
+        participant_origin = self.user_info.get('clientId')
+        if participant_origin not in GENOMIC_CLIENT_IDS:
+            raise BadRequest('Client Id cannot access GenomicScheduling lookup.')
```

Comment:
Can this be a 403 rather than a 400? That way it's clearer it's an unauthorized access issue.

Reply:
Defintiely.